### ISSUE TITLE
Clio in Studio Code

### ISSUE THREAD

**Issue by Mantikor123** [Advance-Technologies-Foundation/clio-docs#4]
At some point, Clio stopped responding to the  Studio Code, we tried reinstalling the plugin, but it didn’t work.
What other solutions could there be?
![not work clio](https://github.com/Advance-Technologies-Foundation/clio-docs/assets/24190423/683af603-0615-42fe-8f22-ba594edff5a4)
![not work clio2](https://github.com/Advance-Technologies-Foundation/clio-docs/assets/24190423/aaf72abe-f6ab-4d28-93b0-e5bb3793d727)

**Comment by GodlyVert** [comment]
![image](https://github.com/Advance-Technologies-Foundation/clio-docs/assets/42441281/fcc21d5d-750c-4a28-8144-f6628f4d3ab2)
Same issue with downloading configuration

**Comment by vladimir-nikonov** [comment]
try do next

open clio settings file. For do it in command promt run command
`clio cfg open`

save youer settings and delete file. rerun clio explorer

**Comment by vladimir-nikonov** [comment]
update you version of clio to 6.0.2.17
<img width="451" alt="image" src="https://github.com/Advance-Technologies-Foundation/clio-docs/assets/26967647/d5edbc46-7963-4b4d-b4ea-a5bef4b41565">

**Comment by RyanFarley** [comment]
I just upgraded to Clio explorer 2.0.53 and Clio 6.0.2.34 and started getting this as well. I cannot see any of the previously registered environments.

If I use clio from command line everything works as usual. I can open the settings config using: 
`clio open-settings`
and see all of the registered environments. They are still there in the file, although Clio Explorer no longer sees them.

In VSCode Clio Explorer does not load any environments. Also, if I use the Clio Explorer menu in VSCode and select "Open settings" nothing opens, no settings file opens as expected. It appears that the Clio Explorer cannot find the settings file and therefore does not load the environments in the file. 

I am on a Mac using latest VSCode. Also, Clio Explorer was working correctly prior to updating (not sure which version I was on previously, but guessing it was likely Clio Explorer 2.0.52)

**Comment by jcollazoMeritus** [comment]
I'm having the same issue on windows, with the same versions: Clio explorer 2.0.53 and Clio 6.0.2.34. 
In Visual Studio Code Clio Explorer does not load any environments

**Comment by vladimir-nikonov** [comment]
@RyanFarley  hello, can you provide your JSON settings file without sensitive information? Just please make sure that after its modification, the error can still be reproduced on it

**Comment by RyanFarley** [comment]
@vladimir-nikonov I have found what the issue is. 

In my appsettings.json file the "ActiveEnvironmentKey" had a value in it that is no longer one of my registered environments. I had a default environment there that I had since removed from the list environments. This is what was preventing the environments from loading. Once I updated the ActiveEnvironmentKey to the name of one of my environments, the list of environments did load as expected. 

To reproduce, update the json and set a value for ActiveEnvironmentKey that is not one of your environments (or delete the environment that is set as the active/default), then reopen VSCode and no environments will load. 

Ryan
